src/vs/workbench/electron-sandbox/desktop.contribution.ts

```diff
@@ -344,6 +344,14 @@ import { IJSONSchema } from 'vs/base/common/jsonSchema';
 				type: 'string',
 				markdownDescription: nls.localize('argv.forceColorProfile', 'Allows to override the color profile to use. If you experience colors appear badly, try to set this to `srgb` and restart.')
 			},
+			'enable-crash-reporter': {
+				type: 'boolean',
+				markdownDescription: nls.localize('argv.enableCrashReporter', 'Allows disabling crash reporting to Appcenter, should restart the app if the value is changed.')
```

Comment:
@deepak1556 I wonder if we need to mention `AppCenter` here, since this may change over time. I would change it to be more abstract and talk about crash reporting but not the endpoint.